### ISSUE TITLE
Suggestion:  Add CombatTimeFu to panel modules

### ISSUE THREAD

**Issue by Cruzix** [shagu/pfUI#163]
I would appreciate if you could add a panel module similar to FuBar_CombatTimeFu, all it does is adding a timer to the panel that automatically starts when entering combat and stops when you get out of combat. 

It can be downloaded here http://www.vanilla-addons.com/dls/fubar-complete-pack/

**Comment by shagu** [comment]
Should the timer reset everytime the combat begins?

**Comment by Cruzix** [comment]
Yes - it should pause once the combat ends and then reset and start a new timer on next combat engage
